### PR TITLE
Add orthoflow

### PR DESCRIPTION
**Pull request by vinisalazar**
Add orthoflow v0.2.0

### CODE REVIEW

**Comment by vinisalazar** [comment]
@BiocondaBot please add label

**Comment by vinisalazar** [comment]
cc @rbturnbull

File: recipes/orthoflow/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - poetry-core >=1.0.0
+    - pip
+  run:
+    - python
+    - snakemake >=7.0.0
```
Comment (mencian) [inline recipes/orthoflow/meta.yaml:29]:
```suggestion
    - snakemake-minimal >=7.0.0
```

**Review by mencian — COMMENTED** [review]
A minor suggestion to use `snakemake-minimal` instead of `snakemake`, otherwise looks good to me!

**Comment by rbturnbull** [comment]
I confirm that I am a maintainer.

**Comment by rbturnbull** [comment]
Thanks @mencian - that's a good idea to depend on `snakemake-minimal` instead.

**Review by mencian — APPROVED** [review]

**Pull request merged**